README.md

```diff
@@ -672,6 +672,15 @@ The API Supports a number of options to configure the read
      </td>
      <td>Read</td>
    </tr>
+   <tr valign="top">
```

Comment:
This should be added to README-template.md, not the README it self (the template was added yesterday. Please rebase the branch)